File: qlty-cli/tests/lang/java/basic.in/functions/SingletonMethodsWithParams.java
import java.util.Arrays;
import java.util.List;
import java.util.stream.Collectors;

class SingletonMethodsWithParams {
  public static List<String> bar(Object dog, Object cat) {
    return Arrays.asList(dog, cat).stream()
        .map(animal -> animal.toString())
        .collect(Collectors.toList());
  }
}

File: qlty-cli/tests/lang/csharp/basic.in/functions/SingletonMethodsWithParams.cs
using System;
using System.Collections.Generic;
using System.Linq;

public class SingletonMethodsWithParams
{
    public static List<string> Bar(object dog, object cat)
    {
        return new List<object> { dog, cat }
            .Select(animal => animal.ToString())
            .ToList();
    }
}